apps/essimporter/importer.cpp

```diff
@@ -349,7 +350,7 @@ namespace ESSImport
 
         writer.setFormat (ESM::SavedGame::sCurrentFormat);
 
-        std::ofstream stream(mOutFile.c_str(), std::ios::binary);
+        boost::filesystem::ofstream stream(boost::filesystem::path(mOutFile.c_str()), std::ios::out | std::ios::binary);
```

Comment:
mOutFile is a string, it shouldn't be necessary to convert it to C string to construct a boost::filesystem::path here either.